packages/hooks/use-css-var/index.ts

```diff
@@ -0,0 +1,43 @@
+import { watch, isRef } from 'vue'
+import type { Ref, WatchStopHandle } from 'vue'
+
+let stopWatchCssVar: WatchStopHandle | null = null
+
+export function useCssVar(
+  vars:
+    | Ref<Record<string, string>>
+    | Record<string, string>,
+  target?: HTMLElement,
```

Comment:
Since the hook i invoked during `setup`, the target could have the possibilities to be a `RefElement`, by the time it's passed into this hook, the value would always be undefined.
I suggest we dispatch the action(attaching vars) at the `onMouned` hook stage so that the target could just be valid here.


```suggestion
target?: Ref<HTMLElement> | HTMLElement.
```